packages/types/package.json

```diff
@@ -21,17 +21,15 @@
   ],
   "scripts": {
     "prepublishOnly": "npm run build",
-    "build": "tsc -p tsconfig.build.json",
-    "dev": "tsc -p tsconfig.build.json --watch",
+    "build": "tsc -p tsconfig.json",
+    "dev": "npm run build -- --watch",
     "clean": "rimraf ./dist",
     "lint": "eslint ."
   },
   "dependencies": {
     "csstype": "3.1.1"
   },
   "devDependencies": {
-    "jest": "^27.4.7",
-    "ts-jest": "^27.1.3",
```

Comment:
❓ shouldn't this change affect the root level package-lock.json ?

Reply:
Correct. Fixing 🙂